### PR TITLE
fix SIMULATE_WEB_RESPONSE not imported #449

### PR DESCRIPTION
**Pull request by wsdwsd0829**
I could not found the correct format for "license as suggested by our bot",  what should I use exactly?

### CODE REVIEW

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please ensure license is correct for ViewController.m:</p>
<pre><code>//
//  ViewController.m
//  Texture
//
//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
//  This source code is licensed under the BSD-style license found in the
//  LICENSE file in the /ASDK-Licenses directory of this source tree. An additional
//  grant of patent rights can be found in the PATENTS file in the same directory.
//
//  Modifications to this file made after 4/13/2017 are: Copyright (c) 2017-present,
//  Pinterest, Inc.  Licensed under the Apache License, Version 2.0 (the "License");
//  you may not use this file except in compliance with the License.
//  You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
</code></</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: examples/ASCollectionView/Sample/ViewController.m [hunk 1]
```diff
@@ -70,8 +70,8 @@ - (void)viewDidLoad
     {
       NSLog(@"ViewController is not nil");
       strongSelf->_data = [[NSArray alloc] init];
-      [strongSelf->_collectionView performBatchUpdates:^{
-        [strongSelf->_collectionView insertSections:[[NSIndexSet alloc] initWithIndexesInRange:NSMakeRange(0, 100)]];
+      [strongSelf->_collectionNode performBatchUpdates:^{
+        [strongSelf->_collectionNode insertSections:[[NSIndexSet alloc] initWithIndexesInRange:NSMakeRange(0, 100)]];
       } completion:nil];
       NSLog(@"ViewController finished updating collectionView");
```
Comment (nguyenhuy) [inline examples/ASCollectionView/Sample/ViewController.m]: Please update the comments here and a couple of lines below to use "collectionNode"

File: examples/ASCollectionView/Sample/ViewController.m [hunk 2]
```diff
@@ -81,7 +81,7 @@ - (void)viewDidLoad
   };
   
   dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(2 * NSEC_PER_SEC)), dispatch_get_main_queue(), mockWebService);
-  dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(2 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
+  dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(4 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
```
Comment (nguyenhuy) [inline examples/ASCollectionView/Sample/ViewController.m]: While I'm not opposing to the change at this line, I'm interested in your reasons.

File: examples/ASCollectionView/Sample/ViewController.m (hunk 1 above)
Comment (wsdwsd0829) [inline examples/ASCollectionView/Sample/ViewController.m]: with SIMULATE_WEB_RESPONSE, compiler will give error saying collectionView undefined, this change will follow the property to use collectionNode

File: examples/ASCollectionView/Sample/ViewController.m (hunk 2 above)
Comment (wsdwsd0829) [inline examples/ASCollectionView/Sample/ViewController.m]: I do not know why need last dispatch at all, current behavior is when click push, the collection node/view will display after 2 second, then pop after another 2s (thus 4 total)
Actually I think we do not need to pop automatically, this change just try to make minimal change

Comment (nguyenhuy) [inline examples/ASCollectionView/Sample/ViewController.m]: To make minimal change, let's revert the change here (i.e change it back to 4 seconds).

File: examples/ASCollectionView/Sample/ViewController.m (hunk 1 above)
Comment (nguyenhuy) [inline examples/ASCollectionView/Sample/ViewController.m]: Yes, I understand the change here. I was suggesting to update the logs to use "collectionNode":
`NSLog(@"ViewController finished updating collectionNode")` and `NSLog(@"ViewController is nil - won't update collectionNode")`. Sorry for the confusion :)

Comment (wsdwsd0829) [inline examples/ASCollectionView/Sample/ViewController.m]: Actually, if we keep using 2s it will cause crash at L115 ASCollectionLayout, because poping VC and layout happen at same time; So I keep 4. 
- (CGSize)collectionViewContentSize
{
  ...
  ASDisplayNodeAssertNotNil(_layout, @"Collection layout state should not be nil at this point")
...

Let me know if it's potential bug and how to fix it.  Thanks.

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please ensure license is correct for ViewController.m:</p>
<pre><code>//
//  ViewController.m
//  Texture
//
//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
//  This source code is licensed under the BSD-style license found in the
//  LICENSE file in the /ASDK-Licenses directory of this source tree. An additional
//  grant of patent rights can be found in the PATENTS file in the same directory.
//
//  Modifications to this file made after 4/13/2017 are: Copyright (c) 2017-present,
//  Pinterest, Inc.  Licensed under the Apache License, Version 2.0 (the "License");
//  you may not use this file except in compliance with the License.
//  You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
</code></</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>
